app/javascript/elm/src/Main.elm

```diff
@@ -612,8 +612,8 @@ view model =
 
 
 viewSearchAsIMove : Bool -> Bool -> Html Msg
-viewSearchAsIMove wasMapMoved isSearchOn =
-    div []
+viewSearchAsIMove wasMapMoved isSearchAsIMoveOn =
+    div [ Html.Attributes.style "position" "absolute", Html.Attributes.style "z-index" "99999" ]
```

Comment:
This as temporary. @gitjul I thought it's better if you can improve and move this to proper css file when styling,  cause I didn't want to mess with the css structure 😉

Reply:
Perfect:) feel free to leave comments for me in the code if you prefer